walkthroughs/howto-cross-ekscluster-appmesh-with-cloudmap/helm/tiller-rbac.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1 
```

Comment:
it is better to skip this and point to eks-charts documentation (as suggested in README.md). These get stale and needs fixing.

Reply:
Well the eks charts does not document installing tiller. we would need this.
We can point them to helm docs but i found it more confusing for folks, as it has too many options

Reply:
Can we improve the documentation under eks-charts instead?

Reply:
@smmallu I'd argue its not really our scope to instruct folks how to install Tiller.  Additionally, helmv3 is out soon and will not require Tiller at all.